src/fragments/lib/logging/common/hub_events/hub_events.mdx

```diff
@@ -0,0 +1,13 @@
+The Amplify Logger send error events through Amplify Hub.  To ensure that errors do not occur when logging, log messages should be validated and follow the best security practices.  Additionally, you should ensure that log messages do not exceed the [Amazon CloudWatch log event size of 256 KB](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html).
```

Comment:
Small tweak: Also - do they not need to import Hub? or does it get imported by default from the singleton?

The Amplify Logger sends errors that occur when using it through Amplify Hub. To ensure that errors do not occur when logging, log messages should be validated and follow the best security practices. Additionally, you should ensure that log messages do not exceed the [Amazon CloudWatch log event size of 256 KB](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html).

Reply:
Text updated.